src/views/css-styles/Checked.jsx

```
@@ -0,0 +1,46 @@
+            <span>
+            {":autofill is a pseudo-class that represents an element that is being autofilled by the browser."}
+            {"However, the most of modern browsers uses !important for autofill styles so you may find some style properties are not working."}
+            </span>
+          </p>
+        </Card>
+        <Card>
+          <h2>Radio button</h2>
+          <div className="flex flex-col gap-2">
+            <div>
+              <input type="radio" name="my-input" id="yes" className="radio-yes peer"/>
+              <label for="yes" className="peer-[.radio-yes]:peer-checked:text-red-500">Yes</label>
```

ESLint

[error] Unknown property 'for' found, use 'htmlFor' instead

(react/no-unknown-property)